api/schema/types/ContributorType.js

```diff
@@ -15,6 +14,9 @@ module.exports = gql`
         permissions: [Permission]
         timeEntries: [TimeEntry]
         rates: [Rate]
+        #The following attributes are calculated and aren't on the database
```

Comment:
Change to `"The following attributes are calculated and aren't on the database"` to expose this as documentation